pkg/capabilities/triggers/mercury_trigger.go

```diff
@@ -24,8 +23,44 @@ var mercuryInfo = capabilities.MustNewCapabilityInfo(
 
 const defaultTickerResolutionMs = 1000
 
+type MercuryTriggerOutput struct {
```

Comment:
This is the same as TriggerEvent, which is a generic struct to support all triggers, not only Mercury. Why do we need a separate one here?